layouts/security/single.html

```diff
@@ -0,0 +1,163 @@
+                  <span><b>{{ i18n "Download:" }}</b> <a href="https://repo.almalinux.org/almalinux/RPM-GPG-KEY-AlmaLinux-9">AlmaLinux</a></span>
+                  <span><b>{{ i18n "Download:" }}</b> <a href="https://pgp.mit.edu/pks/lookup?op=index&search=0xD36CB86CB86B3716">pgp.mit.edu</a></span>
+                  <div class="container-locks" style="display: flex; justify-content: flex-start; align-items: center; margin-top: 15px;">
+                    <img class="Photo__Lock" style="width: 40px; height: 40px;" src="/images/alma-fingerprint-A.svg" />
+                    <div style="display: flex; flex-direction: column;">
+                      <p style="margin-left: 10px; font-family: monospace; font-size: 14px; color: #fff;">
+                        BF18 AC28 7617 8908 D6E7 1267 D36C B86C B86B 3716
+                      </p>
+                    </div>
+                  </div>
+                </div>
+                <div class="itemAl_02" style="padding-left: 20px; width: 608px; display: flex; flex-direction: column;">
+                  <h3>AlmaLinux OS 8</h3>
+                  <span><b>rsa4096/488FCF7C3ABB34F8 (2021-01-12):</b></span>
+                  <span>AlmaLinux  &lt;packager@almalinux.org&gt;</span>
+                  <span><b>{{ i18n "Location:" }}</b> /etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux</span>
+                  <span><b>{{ i18n "Download:" }}</b> <a href="https://repo.almalinux.org/almalinux/RPM-GPG-KEY-AlmaLinux">AlmaLinux</a></span>
+                  <span><b>{{ i18n "Download:" }}</b> <a href="https://pgp.mit.edu/pks/lookup?op=index&search=0x488FCF7C3ABB34F8">pgp.mit.edu</a></span>
+                  <div class="container-locks" style="display: flex; justify-content: flex-start; align-items: center; margin-top: 15px;">
+                    <img class="Photo__Lock" style="width: 40px; height: 40px;" src="/images/alma-fingerprint-A.svg" />
+                    <div style="display: flex; flex-direction: column;">    
+                      <p style="margin-left: 10px; font-family: monospace; font-size: 14px; color: #fff;">
+                        5E9B 8F56 17B5 066C E920 57C3 488F CF7C 3ABB 34F8
+                      </p>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </section>
+          </div>
+        </div>
+        <div class="SecurityMailingList-container" style="display: flex; flex-direction: column; padding: 40px 20px;">
+          <h2 class="pb-2 text-center border-bottom" style="border-bottom-color: #14598a!important;">Security Mailing List</h2>
+          <div class="al-article-content pb-3 al-wysiwyg" style="padding: 0px!important;">
+            <section>
+              <p>
+                {{ i18n "To receive security updates and errata, users can sign up for the " }}
+                <a href="https://lists.almalinux.org/mailman3/lists/" style="color: #f1f8ff;">{{ i18n "AlmaLinux Security Mailing List" }}</a>.
```

Comment:
```suggestion
                <a href="https://lists.almalinux.org/mailman3/lists/security.lists.almalinux.org/" style="color: #f1f8ff;">{{ i18n "AlmaLinux Security Mailing List" }}</a>.
```

Should we link straight to the security list instead of the mailing list index?

Reply:
@jonathanspw yes, nice!